packages/react-sketch-canvas/src/Canvas/index.tsx

```diff
@@ -291,13 +311,31 @@ release drawing even when point goes out of canvas */
     );
   }, [paths]);
 
+  // avoid pen from scrolling if scrollOnTouch
+  useEffect(() => {
+    const listener = function(
+      this: HTMLDivElement,
+      event: TouchEvent
+    ): void {
+      if (currentPointerType.current === "pen") {
+        event.preventDefault();
+        event.stopPropagation();
+      }
+    };
+
+    if (scrollOnTouch) {
+      canvasRef.current?.addEventListener("touchstart", listener, { passive: false });
+    }
+    return () => canvasRef.current?.removeEventListener("touchstart", listener);
+  }, []);
```

Comment:
As the dependency array is empty, if the developer changes scrollOnTouch dynamically, this function will fail.